### PR TITLE
[CS] Add missing method scope and fix function comment style

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: lib/Doctrine/ODM/MongoDB/Types/DateType.php [hunk 1]
```diff
@@ -66,7 +66,6 @@ public static function getDateTime($value)
         return $datetime;
     }
 
-    // @todo fix typing for $microseconds
```
Comment (carusogabriel) [inline lib/Doctrine/ODM/MongoDB/Types/DateType.php]: This one was auto removed. Should we allow the `@todo` annotation in our CS?

File: tests/Doctrine/ODM/MongoDB/Tests/Mapping/AnnotationDriverTest.php [hunk 2]
```diff
@@ -12,7 +12,6 @@
 
 class AnnotationDriverTest extends AbstractMappingDriverTest
 {
-    // @TODO: This can be a generic test for all drivers
```
Comment (carusogabriel) [inline tests/Doctrine/ODM/MongoDB/Tests/Mapping/AnnotationDriverTest.php]: This one was auto removed. Should we allow the `@TODO` annotation in our CS?

File: tests/Doctrine/ODM/MongoDB/Tests/UnitOfWorkTest.php [hunk 3]
```diff
@@ -116,8 +116,7 @@ public function testRegisterRemovedOnNewEntityIsIgnored()
     }
 
 
-    /* Operational tests */
-
+    /** Operational tests */
```
Comment (carusogabriel) [inline tests/Doctrine/ODM/MongoDB/Tests/UnitOfWorkTest.php]: Is this a false-positive forcing to be a function comment?

Comment (malarzm) [reply tests/Doctrine/ODM/MongoDB/Tests/UnitOfWorkTest.php]: I think yes

File: lib/Doctrine/ODM/MongoDB/Types/DateType.php (hunk 1 above)
Comment (alcaeus) [reply lib/Doctrine/ODM/MongoDB/Types/DateType.php]: Yes, I believe this is a good idea. There will be a few of those left in while we work in 2.0. We should however remove the exception before tagging 2.0.

File: tests/Doctrine/ODM/MongoDB/Tests/Functional/CustomTypeTest.php [hunk 4]
```diff
@@ -51,7 +51,7 @@ class DateCollectionType
 {
     use ClosureToPHP;
 
-    // Note: this method is called by PersistenceBuilder
+    /** Note: this method is called by PersistenceBuilder */
```
Comment (alcaeus) [inline tests/Doctrine/ODM/MongoDB/Tests/Functional/CustomTypeTest.php]: This should either be a proper docblock or dropped completely: the information where it's called from can change any time and can easily be inferred from the IDE. If we want to add a comment (which might be better off in the base class as it applies to all types) we should explain what the method is expected to do.

File: tests/Doctrine/ODM/MongoDB/Tests/Functional/CustomTypeTest.php [hunk 5]
```diff
@@ -90,7 +90,7 @@ public function convertToPHPValue($value)
         return $value;
     }
 
-    // Note: this method is never called
+    /** Note: this method is never called */
```
Comment (alcaeus) [inline tests/Doctrine/ODM/MongoDB/Tests/Functional/CustomTypeTest.php]: Same as above.

File: tests/Doctrine/ODM/MongoDB/Tests/Functional/QueryTest.php [hunk 6]
```diff
@@ -389,7 +389,7 @@ public function testQueryWhereInReferenceId()
         $this->assertSame($expected, $qb->getQuery()->debug('query'));
     }
 
-    // search for articles that have the "pet" tag in their tags collection
+    /** search for articles that have the "pet" tag in their tags collection */
```
Comment (alcaeus) [inline tests/Doctrine/ODM/MongoDB/Tests/Functional/QueryTest.php]: TBH, the value of this comment is 0 - I can see what it's doing in the code. I suggest removing this kind of comments entirely.

File: tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH1294Test.php [hunk 7]
```diff
@@ -48,7 +48,7 @@ class GH1294User
     /** @ODM\Field(type="string") */
     public $name = false;
 
-    // Return the identifier without triggering Proxy initialization
+    /** Return the identifier without triggering Proxy initialization */
```
Comment (alcaeus) [inline tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH1294Test.php]: The comment refers to an implementational detail of proxies, which is irrelevant in the base class. Again, I suggest dropping this comment and the ones like it.

File: tests/Documents/Functional/VirtualHostDirective.php [hunk 8]
```diff
@@ -48,8 +48,7 @@ public function setRecId($value = null)
         $this->recId = $value;
     }
 
-    /* Added automatically 2010-08-03 */
-
+    /** Added automatically 2010-08-03 */
```
Comment (alcaeus) [inline tests/Documents/Functional/VirtualHostDirective.php]: This is virtually useless.

File: tests/Doctrine/ODM/MongoDB/Tests/UnitOfWorkTest.php (hunk 3 above)
Comment (alcaeus) [reply tests/Doctrine/ODM/MongoDB/Tests/UnitOfWorkTest.php]: This is also just a useless comment that we could remove.

File: lib/Doctrine/ODM/MongoDB/Types/DateType.php (hunk 1 above)
Comment (carusogabriel) [reply lib/Doctrine/ODM/MongoDB/Types/DateType.php]: I'll move these `@todo` annotations inside the method scope instead of keeping them as annotations :)

**Comment by carusogabriel** [comment]
@alcaeus Comments changes available in 4ff476e

**Pull request merged**